src/Tizen.Applications.WidgetControl/Tizen.Applications/WidgetControl.cs

```diff
@@ -77,9 +80,11 @@ public Bundle GetContent()
             /// <since_tizen> 3 </since_tizen>
             /// <param name="content">Content to be changed.</param>
             /// <param name="force"> True if you want to update your widget even if the provider is paused, otherwise false.</param>
+            /// <feature>http://tizen.org/feature/shell.appwidget</feature>
             /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
             /// <exception cref="InvalidOperationException">Thrown in case of failed conditions.</exception>
             /// <exception cref="UnauthorizedAccessException">Thrown when an application does not have the privilege to access this method.</exception>
```

Comment:
I'd like to recommend the following sentences:

/// Thrown when an application does not have the required privilege to access this method.
/// Thrown when an application does not have the required privileges to access this method.
/// Thrown when an application does not have the required privilege(s) to access this method.

How about using one of three sentences? In my opinion, the 2nd one is good.